geth/wrapper.py

```diff
@@ -154,8 +155,8 @@ def construct_popen_command(data_dir=None,
 
     if ipc_api is not None:
         raise DeprecationWarning(
-            "The ipc_api flag has been deprecated.  The ipc API is now on by "
-            "default.  Use `ipc_disable=True` to disable this API"
+            'The ipc_api flag has been deprecated.  The ipc API is now on by '
```

Comment:
**nit:** These kind of just muddy up the commit history. If we could avoid this change that would be nice 👀. We may even add `black` to this repo at some point and this would just revert back to `"`.